### DISCUSSION TITLE
i got an error Line2D.set()

### DISCUSSION THREAD

**Discussion by yvr354** [open-edge-platform/anomalib#1993] [Q&A]
i am using old anomalib in ubunto 22.04 i got an error , AttributeError: Line2D.set() got an unexpected keyword argument 'figsize', can anyone help me how to solve this .

**Comment by samet-akcay** [comment]
Hi @yvr354, can you provide more information, like what was the command you ran, what was you config file etc.

**Reply by yvr354** [reply]
dataset:
    name: custom
    format: folder
    root: datasets/bumpers_data
    normal_dir: GOOD # name of the folder containing normal images.
    abnormal_dir: test_NEW #broken_val #Reject_1 #Reject # name of the folder containing abnormal images.
    task: segmentation # classification or segmentation
    mask_dir: ground_truth_NEW #optional
    normal_test_dir: null # optional
    extensions: null
    split_ratio: 0.2 # normal images ratio to create a test split
    seed: 0
    image_size: 800
    train_batch_size: 8
    eval_batch_size: 8
    num_workers: 8
    normalization: imagenet # data distribution to which the images will be normalized
    test_split_mode: from_dir # options [from_dir, synthetic]
    val_split_ratio: 0.4 # fraction of train/test images held out for validation (usage depends on val_split_mode)
    transform_config:
        train: null
        eval: null
    val_split_mode: from_test # determines how the validation set is created, options [same_as_test, from_test]
    tiling:
        apply: false
        tile_size: null
        stride: null
        remove_border_count: 0
        use_random_tiling: False
        random_tile_count: 16

model:
  name: padim
  backbone: resnet18
  pre_trained: true
  layers:
    - layer1
    - layer2
    - layer3
  normalization_method: min_max # options: [none, min_max, cdf]

metrics:
  image:
    - F1Score
    - AUROC
  pixel:
    - F1Score
    - AUROC
  threshold:
    method: adaptive #options: [adaptive, manual]
    manual_image: null
    manual_pixel: null

visualization:
  show_images: False # show images on the screen
  save_images: True # save images to the file system
  log_images: True # log images to the available loggers (if any)
  image_save_path: null # path to which images will be saved
  mode: full # options: ["full", "simple"]

project:
  seed: 42
  path: ./results_fuck

logging:
  logger: [ csv] # options: [comet, tensorboard, wandb, csv] or combinations.
  log_graph: true # Logs the model graph to respective logger.

optimization:
  export_mode: onnx  # options: torch, onnx, openvino null

# PL Trainer Args. Don't add extra parameter here.
trainer:
  enable_checkpointing: true
  default_root_dir: null
  gradient_clip_val: 0
  gradient_clip_algorithm: norm
  num_nodes: 1
  devices: 1
  enable_progress_bar: true
  overfit_batches: 0.0
  track_grad_norm: -1
  check_val_every_n_epoch: 1 # Don't validate before extracting features.
  fast_dev_run: false
  accumulate_grad_batches: 1
  max_epochs: 1
  min_epochs: null
  max_steps: -1
  min_steps: null
  max_time: null
  limit_train_batches: 1.0
  limit_val_batches: 1.0
  limit_test_batches: 1.0
  limit_predict_batches: 1.0
  val_check_interval: 1.0 # Don't validate before extracting features.
  log_every_n_steps: 5
  accelerator: auto # <"cpu", "gpu", "tpu", "ipu", "hpu", "auto">
  strategy: null
  sync_batchnorm: false
  precision: 32
  enable_model_summary: true
  num_sanity_val_steps: 0
  profiler: null
  benchmark: false
  deterministic: false
  reload_dataloaders_every_n_epochs: 0
  auto_lr_find: false
  replace_sampler_ddp: true
  detect_anomaly: false
  auto_scale_batch_size: false
  plugins: null
  move_metrics_to_cpu: false
  multiple_trainloader_mode: max_size_cycle



#python tools/train.py --config src/anomalib/models/padim/padim.yaml

**Comment by yvr354** [comment]
![anomalib_gcp_error](https://github.com/openvinotoolkit/anomalib/assets/68755664/92307d2a-f4c9-45f1-bcc5-505fdf9138d0)
 
python tools/train.py --config src/anomalib/models/padim/padim.yaml
